kolibri/plugins/facility/assets/src/constants.js

```diff
@@ -10,6 +10,7 @@ export const PageNames = {
   DATA_EXPORT_PAGE: 'DATA_EXPORT_PAGE',
   FACILITY_CONFIG_PAGE: 'FACILITY_CONFIG_PAGE',
   ALL_FACILITIES_PAGE: 'ALL_FACILITIES_PAGE',
+  ManageSyncSchedule: 'ManageSyncSchedule',
```

Comment:
We should follow the same convention for this constant as the one above - use UPPERCASE_SNAKECASE.

It also seems like we are missing one for `EDIT_SYNC_SCHEDULE`?